system/lib/libc/musl/include/unistd.h

```diff
@@ -258,7 +258,11 @@ pid_t gettid(void);
 
 #define _POSIX_VDISABLE         0
 
+#if defined(__EMSCRIPTEN__) && !defined(_REENTRANT) /* XXX Emscripten doesn't always support pthreads */
+#define _POSIX_THREADS          -1
```

Comment:
Is -1 some kind of POSIX convention? I tried to look for it, and the docs I found all suggest "is defined when threads are enabled", so I'd expect us to not define it in that case? That seems to be what search results on github expect:

https://github.com/search?q=%22_POSIX_THREADS%22&type=code

Reply:
Yes, I noticed that we used -1 elsewhere in this file to mean "no available".  And I was confused too.  But it looks like -1 does mean "not available".

See the discussion in the original PR: #9452

As an example, here is hurd declaring that it doesn't support stuff: https://codebrowser.dev/glibc/glibc/sysdeps/mach/hurd/bits/posix_opt.h.html#163

Reply:
Interesting... not sure how that works with those github results that do check for definition..? Perhaps this just isn't depended on in practice? Anyhow, it seems low risk and worst case we can revert.